design/service-apis-implementation.md

```diff
@@ -0,0 +1,229 @@
+# Service-APIs implementation design
+
+Status: Accepted
```

Comment:
Well, I figured that if we merge it, it's accepted. Otherwise, this will get closed. Traditionally, we've just not updated this field, and everything is a draft, which is also not ideal. :)

Reply:
We could just drop the field, and assume anything that's been merged is accepted; anything still a PR is draft. Not a big deal though.

Reply:
The original idea was to submit ideas here without them being "accepted" yet, to gather thoughts, etc. We've been waiting to commit the design until it's accepted in a way, so removing the field might better follow how we're implementing features.

Reply:
Yeah, removing the field may make more sense, if it's merged, it's accepted. If we change things like that, we'll need to:
- update the template and remove the field
- update CONTRIBUTING.md/ other policy docs
- update the old PRs to remove the field.